### PR TITLE
Update README to address terminology confusion.

### PR DESCRIPTION
**Pull request by steven-collins-omega**
In passing address official open-source Razee support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +30,10 @@ This diagram provides an overview of the relationships between the various entit
 
 A typical SatCon workflow might proceed roughly as follows:
 
-1. **Register a cluster to SatCon.**  This makes the cluster known to SatCon, and the API call will return the URL for a kube yaml file which can be applied to the actual cluster.  This yaml will deploy and configure the SatCon (razee) agent, which in turn will connect back to SatCon to begin managing the cluster.
-1. **Add the cluster to one or more groups.**  If no group exists within the org, you will first need to create a group.  You can then add your cluster to the group.
+1. **Register a cluster to SatCon.**  This makes the cluster known to SatCon, and the API call will return the URL for a Kubernetes yaml file which can be applied to the actual cluster.  This yaml will deploy and configure the SatCon (razee) agent, which in turn will connect back to SatCon to begin managing the cluster.
```
Comment (ScarletTanager) [inline README.md:33]: Change `This yaml` to `Applying this yaml via e.g. kubectl` or similar.

File: README.md [hunk 2]
```diff
@@ -26,10 +30,10 @@ This diagram provides an overview of the relationships between the various entit
...
+1. **Add the cluster to one or more groups.**  If no group exists in the IBM Cloud account, you will first need to create a group.  You can then add your cluster to the group.
```
Comment (ScarletTanager) [inline README.md:34]: I would prefer, `within the organization (i.e. the IBM Cloud account)` - I think that we should make sure to use both terms everywhere we use one or the other for now.

**Review by ScarletTanager — CHANGES_REQUESTED** [review]
A couple of tweaks requested.

**Comment by steven-collins-omega** [comment]
@ScarletTanager tweaks made along with a couple of other minor improvements.

**Review by ScarletTanager — APPROVED** [review]
Wazinga!

**Pull request merged**